content/en/Debugging.md

```diff
@@ -0,0 +1,32 @@
+---	 
+title: Debugging	 
+status: Completed	 
+category: Concept	 
```

Comment:
For now, [Netlify build fails](https://app.netlify.com/sites/cncfglossary/deploys/62075abab5c1c00008d7e597#L67).

> failed to unmarshal YAML: yaml: found character that cannot start any token

I guess that this is because there are some tab characters (`\t`) in this file:

```suggestion
---
title: Debugging
status: Completed
category: Concept
```

And, currently most filenames of markdown files in [`content/en/`](https://github.com/cncf/glossary/tree/main/content/en) start with lower case, except `TLS` which is a acronym.

So, please consider to rename this file to `debugging.md` ! 😊

Reply:
@Krishnamohan-Yerrabilli 

Please update the PR according to @jihoon-seo 's suggestions to resolve build failure issue :)
(renaming file, too)

Reply:
My class is going on 
After my college is I will fix it thank you :)

Reply:
As per @jihoon-seo's suggestions changes have been made.